seed_data/management/programs.json

```diff
@@ -1,273 +1,221 @@
 [
```

Comment:
this is newly-generated program data to be used as seed data. user data was similarly generated and updated